plugins/PrivacyManager/PrivacyManager.php

```diff
@@ -601,4 +604,36 @@ public static function getUserIdSalt()
         }
         return $salt;
     }
+
+    public function renderPrivacyPolicyLinks(&$out)
+    {
+        $settings = new SystemSettings();
+
+        if (!$this->shouldRenderFooterLinks($settings)) {
+            return;
+        }
+
+        $view = new View('@PrivacyManager/footerLinks.twig');
+        $view->privacyPolicyUrl = $settings->privacyPolicyUrl->getValue();
+        $view->termsAndCondition = $settings->termsAndConditionUrl->getValue();
+        $out .= $view->render();
+    }
+
+    private function shouldRenderFooterLinks(SystemSettings $settings)
+    {
+        $module = Common::getRequestVar('module', false);
+        if ($module == 'Login') {
+            return true;
+        }
```

Comment:
Actually that check is redundant, as all users on the login pages are `anonymous`. But guess we could leave it there for a better understanding